backend/src/test/java/com/hackner/musicompass/service/RecommendationServiceTest.java

```diff
@@ -141,7 +142,98 @@ public void setRecommendationTags(){
         verify(artistMongoDb).save(artistAfter);
         assertThat(actual, equalTo(recommendationTagsAfter));
     }
-/*
+
     @Test
-    @DisplayName("Get Artist Recommendation which is not available")*/
+    @DisplayName("Request Artist Recommendation with tag combination which is not available")
+    public void getArtistRecommendationForNotExistingTagCombination(){
+        //GIVEN
+        List<String> gender = List.of("Female");
+        List<String> genres = List.of("Jazz", "Rock");
+        List<String> roles = List.of("Singer");
+        RecommendationRequestDto recommendationRequestDto = RecommendationRequestDto.builder()
+                .gender(gender)
+                .genres(genres)
+                .roles(roles).build();
+        when(artistMongoDb.findAll()).thenReturn(getArtistList());
+
+        //WHEN
+        String actual = recommendationService.getArtistRecommendation(recommendationRequestDto);
+
+        //THEN
+        assertThat(actual, is("error-not-found"));
```

Comment:
Do not use special strings for status! This is really bad and just asking for trouble.

Reply:
fixed, due to proper error handling